.github/workflows/release.yaml

```diff
@@ -80,13 +96,46 @@ jobs:
           }
         shell: Rscript {0}
 
-      - name: Release
+      - name: cross compile rust object files for another target arch
+        if: matrix.config.target != ''
+        env:
+          RPOLARS_BUILD_COMMAND_BASE: "cargo build --target=${{ env.RUNNER_TARGET }}"
+        run: |
+          pushd src
+          make -f Makevars "./rust/target/${{ env.RPOLARS_PROFILE }}/libr_polars.a"
```

Comment:
@eitsupi  All likely fine, but I fail to understand the exact implications of calling Makevars not via R install. Can you explain what happens when Makevars is passed this path as arg? in  `make -f Makevars "./rust/target/${{ env.RPOLARS_PROFILE }}/libr_polars.a"` What variable name does it become in Makevars? Why is it needed? why is it a path without a target folder?

Reply:
> Can you explain what happens when Makevars is passed this path as arg?

This is simply the usage of the `make` command.
A command `make foo/bar` calls the target named `foo/bar` defined in the Makefile, right?
Here we should specify `.../libr_polars.a` because we need to build `.../libr_polars.a` defined in the Makefile.